src/care/ProfilePlugin.h

```diff
@@ -0,0 +1,30 @@
+#ifndef _CARE_ProfilePlugin_H_
+#define _CARE_ProfilePlugin_H_
+
+#include "RAJA/util/PluginStrategy.hpp"
+
+namespace care{
+
+   class ProfilePlugin : public RAJA::util::PluginStrategy
+   {
+      public:
+         ProfilePlugin();
```

Comment:
Best practice if there is nothing in the body of the constructor is to specify `= default`.

e.g. `ProfilePlugin() = default;`

Then you don't have to define an empty constructor in the .cpp file.